osdk/tests/commands/run.rs

```diff
@@ -67,6 +67,10 @@ mod qemu_gdb_feature {
 
     #[test]
     fn basic_debug() {
+        if std::env::var("INTEL_TDX").is_ok() {
+            println!("Test skipped because TDX is enabled");
```

Comment:
`println!` won't be visible in Cargo tests if the tester don't do `cargo test -- --nocapture`. So just put a comment here describing why.